release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -40,6 +40,43 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-7-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-7-complex-root-devices"]
+==== Complex root tooling for LUKS and RAID devices
+
+Disk tooling enhancements now support complex root devices in bare metal deployments. For example, software RAID improvements now allow for more robust nodes that can still reboot in the event of a drive failure. And support for multiple LUKS root devices means that you now have added capabilities such as using the Local Storage Operator in file mode, having your non-rootfs data encrypted at rest, and use of third-party software that requires local persistent storage.
+
+In addition, support for complex root devices allows:
+
+* New Ignition-native mechanism for configuring root filesystem LUKS. This means that provisioning will fail if the old `/etc/clevis.json` is specified in the Ignition config.
+* Support for LUKS encryption of additional data partitions.
+* Support for boot disk mirroring, except on s390x.
++
+[NOTE]
+====
+On any unencrypted s390x nodes that were originally installed on {product-title} 4.3 or later, the first boot into {product-title} 4.7 takes considerable time proportional to the size of the root filesystem. This is due to a disk format conversion when nodes are first booted into {product-title} 4.7. To prevent data loss, do not interrupt the conversion process.
+====
+
+[id="ocp-4-7-bootupd"]
+==== Update the bootloader using bootupd
+
+With bootupd, {op-system} users now have access to a cross-distribution, system-agnostic OS update tool that manages firmware and boot updates in UEFI and legacy BIOS boot modes that run on modern architectures.
+
+[id="ocp-4-7-rhcos-rhel-8-3-packages"]
+==== {op-system} now supports RHEL 8.3
+
+{op-system} is now using Red Hat Enterprise Linux (RHEL) 8.3 packages. {product-title} 4.6 and below will stay with RHEL 8.2 packages. This is relevant for hardware support, new NetworkManager features, and so on.
```

Comment:
@travier - This is based on your [comment](https://github.com/openshift/openshift-docs/issues/26801#issuecomment-746394196). Does this look OK to you?